apps/condo/domains/notification/ticketTemplates.js

```diff
@@ -0,0 +1,153 @@
+const get = require('lodash/get')
```

Comment:
Please add TODO, to remove this file in a future or don’t forget to remove it

Reply:
I'm sure Alex will remove it when his translations PR will be rebasing to master.